collect_app/src/main/java/org/odk/collect/android/utilities/ColorPickerDialog.java

```diff
@@ -44,6 +44,8 @@
         void colorChanged(String key, int color);
     }
 
+    // scale factor for the dialog, with the height of 90% of screen height
```

Comment:
This could be a doc comment, which would allow pop-up help in an IDE.
/** ... */